packager/VisibilityChecker.cc

```diff
@@ -789,30 +799,34 @@ class VisibilityCheckerPass final {
                 core::Context ctx{gs, core::Symbols::root(), f.file};
                 VisibilityCheckerPass pass{ctx, gs.packageDB().getPackageInfo(pkgName)};
                 ast::TreeWalk::apply(ctx, pass, f.tree);
-                resultq->push(make_pair(f.file, std::move(pass.packageReferences)), 1);
+                resultq->push(
+                    ThreadResult{f.file, std::move(pass.packageReferences), std::move(pass.symbolsReferenced)}, 1);
```

Comment:
Not sure whether it's worth making these fields have the same tense (either switch to `packagesReferenced` or `symbolReferences`)